### PR TITLE
upgraded issue template to issue forms

### LINKED ISSUE #8: Upgrade our issue templates to use GitHub issue forms ✍️
# Introduction

GitHub has recently rolled out a [public beta for their issue forms](https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/) feature. This would allow you to create interactive issue templates and validate them 🤯.

Appwrite currently uses the older [issue template format](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates). Your task is to create **GitHub issue forms** for this repository. Please use [Appwrite's issue templates](https://github.com/appwrite/appwrite/tree/master/.github/ISSUE_TEMPLATE) as a reference for this PR.

### Tasks summary:

- [ ] Fork & clone this repository
- [ ] Prepare **bug report** issue form in `.github/ISSUE_TEMPLATE/bug.yaml`
- [ ] Prepare **documentation** issue form in `.github/ISSUE_TEMPLATE/documentation.yaml`
- [ ] Prepare **feature request** issue form in `.github/ISSUE_TEMPLATE/feature.yaml`
- [ ] Push changes to **master** and test issue forms on your fork
- [ ] Submit pull request

If you need any help, reach out to us on our [Discord server](https://discord.gg/GSeTUeA).

Are you ready to work on this issue? 🤔 Let us know, and we will assign it to you 😊

**Happy Appwriting!**

### PR DESCRIPTION
**Pull request by vj-codes**
Fixes #8

### CODE REVIEW

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Hey, we have few more changes required. For example you can ask for the library version, php version and code of conduct on all the templates. We have a first approved PR please take that as reference
https://github.com/utopia-php/framework/pull/28

**Comment by vj-codes** [comment]
@lohanidamodar done, please review again
Thankss in advance:)

**Comment by lohanidamodar** [comment]
@vj-codes it seems the form has some errors. Please correct those. Make sure to test it on your repository.
image

**Comment by vj-codes** [comment]
@lohanidamodar done!

File: .github/ISSUE_TEMPLATE/feature.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: "🚀 Feature"
+description: "Submit a proposal for a new feature"
+title: "🚀 Feature: "
+labels: [feature]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out our bug report form 🙏
```
Comment (Meldiron) [inline .github/ISSUE_TEMPLATE/feature.yaml:9]: Please update this part as showed in our template. We cannot thank for bug report in feature request.

File: .github/ISSUE_TEMPLATE/documentation.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: "📚 Documentation"
+description: "Report an issue related to documentation"
+title: "📚 Documentation: "
+labels: [documentation]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out our bug report form 🙏
```
Comment (Meldiron) [inline .github/ISSUE_TEMPLATE/documentation.yaml:9]: Please update this part as showed in our template. We cannot thank for bug report in documentation update.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Requested 2 tiny changes

**Comment by christyjacob4** [comment]
@vj-codes  closing this PR for now since there has been no activity in quite a long time. Feel free to reopen it when you're ready to start working on it once again.

**Pull request closed without merging**